nectar/src/swap.rs

```diff
@@ -19,7 +19,7 @@ pub use self::comit::{hbit, herc20};
 pub use crate::database::Database;
 use time::OffsetDateTime;
 
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, strum_macros::Display)]
```

Comment:
Are you aware that the generated `Display` implementation will only print the variant name?

Reply:
Yes, the way I am using it is actually for printing the name for reporting to sentry. I was not planning to report the complete params. It should not be necessary. 
I think that should be fine for now, can optimize this later, what do you think?

Reply:
Yes, I just wasn't sure if you were fully aware of what this does :)